start_server.sh

```diff
@@ -6,12 +6,12 @@ export SD_URL=http://127.0.0.1:7860
 # Check if the desired environment exists
 if [ ! -d "server_env" ]; then
   # Create the environment if it doesn't exist
-  python -m venv server_env
+  python3 -m venv server_env
```

Comment:
python3 is not a recognized command on my system.

Reply:
That might be, but it is up to your settings.
For example, on my remote machine - python is linked to a 2.7 version ( as in many Linux systems by default, I think ).
<img width="244" alt="image" src="https://user-images.githubusercontent.com/42819131/223743094-db683a3c-ffe9-4289-ac6b-accea904fa05.png">

And, as I think, this is the most common behavior, so I suggest specifying the python version ( to avoid any duality ).
By the way, in macOs kickstarter you use an exact command with specific versions:
https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/blob/bca743dd17c77f3b3aa42c2da2829ce19618d0d9/start_sesrver_MacOS.sh#L9

But, anyway, you are right in this detail. I think I can change this script to check python interpreter version at the beginning of execution and show the error message if you have an old python version.